newsfragments/2211.removal.rst

```diff
@@ -0,0 +1,4 @@
+``trio.MultiError`` has been removed in favor of the built-in ``BaseExceptionGroup``
```

Comment:
We usually don't outright remove things, but deprecate them, and give users a few months or a few release to use the new thing. Is it possible in that case?

Reply:
Oh, just noticed this was discussed in chat: https://gitter.im/python-trio/general?at=61f0909d7842bd3ca96472f5